### DISCUSSION TITLE
Multiple images on the viewer: reisizing?

### DISCUSSION THREAD

**Discussion by DevinSmithWork** [openseadragon/openseadragon#2267] [Q&A] [answered]
I'm working on a simple multi-image viewer, using collection mode. The two tileSources are different widths, so by default the narrower one scales up to match the width of the wider one.

However, I want to keep the _heights_ of the images the same instead of the _widths_. I'm a little confused about how to set the default width/height of each tileSource in the collection so they display this way.

Any help would be appreciated!

**Comment by iangilman** [comment, marked as answer]
Collection mode is pretty rudimentary, just if you want something quick and dirty. I believe what it does is arrange a bunch of squares and then center the images inside of those squares. 

To have more control, I recommend using full multi image: 

https://openseadragon.github.io/examples/multi-image/

... And arranging them however you'd like. 

Here's the code that collection mode uses, in case it gives you some ideas: 

https://github.com/openseadragon/openseadragon/blob/a111d0f616c85bae377aca191ff44ec8261f7285/src/world.js#L321

Let me know if you need any more pointers!

**Reply by DevinSmithWork** [reply]
Thank you! I've got the full multi-image + arrange settings going.
